ee/modules/025-static-routing-manager/crds/routingtables.yaml

```diff
@@ -100,6 +100,11 @@ spec:
                   A set of labels that determine the nodes on which the desired routes should be created.
                 additionalProperties:
                   type: string
+                  pattern: "^[a-zA-Z0-9][a-zA-Z0-9-_.]*[a-zA-Z0-9]$|^[a-zA-Z0-9]$|^$"
+                  description: |
+                    The value must be a valid label value: it can contain letters (a-z, A-Z), numbers (0-9), dashes (-), underscores (_), and dots (.),
+                    must start and end with an alphanumeric character (a-z, A-Z, 0-9), and must not contain '*'.
+                    An empty string is allowed to match nodes with the label present regardless of its value.
```

Comment:
```suggestion
```
All theese words are in pattern above.
Also, there must be doc-ru version and there is a validation for this, but it didn't work for some reason 🤷‍♂️ .